src/serving-form.tsx

```diff
@@ -1,14 +1,46 @@
 import React, { Component } from 'react'
+import axios from 'axios'
 import FormCell from './components/form-cell'
 
+const style = {
+  submitStyle: {
+    width: '55%',
+    height: '30px',
+    marginTop: '20px',
+    color: '#fff',
+    background: '#282c34',
+    borderColor: '#fff',
+    borderRadius: 20,
+    borderWidth: 1
+  }
+};
+
 export default class ServingForm extends Component {
```

Comment:
Does this Component need to be typed? `Component<Props>`

Reply:
I'll probably make all the components typed when I figure out the Props that I want for them.